### DISCUSSION TITLE
Kelp Wallet | Wallet "In Review"  Possible to expedite

### DISCUSSION THREAD

**Discussion by edbishop** [WalletConnect/walletconnect-monorepo#2486] [Developer Support] [answered]
Wanted to ask if its possible to expedite our wallet review.  We have a target date of June 5 to launch our project and would like to have the ability to list the Kelp wallet in the registry by then so our users can easily find the wallet when using the mobile browser on our dApp.

**Comment by edbishop** [comment]
Hi @finessevanes Am I able to point my issue to the devs/support team to get this looked at?  Looking to get some sort of estimation on when our wallet can be in review.  We are launching June 5 and would love to have our wallet in the registry for launch.

**Comment by finessevanes** [comment]
@edbishop Hey, It was rejected yesterday. I'm getting an error that the network is not supported.
<img width="520" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/976bbc2a-2ea8-452e-96dc-4bdc828d2602">

**Reply by edbishop** [reply]
@finessevanes  We are currently only supporting BSC. Looks like you are trying to connect via ETH. Pretty sure that the explorer application we filled in states BSC only.

**Reply by finessevanes** [reply]
Do you have a dapp I can test with?

**Comment by edbishop** [comment]
Yes. We have a presale dApp.

presale.kelp.org

I can send you some BNB/BUSD to test with. It’s on mainnet but happy to
send some to the Kelp wallet to test it out.

Just let me know!

On Fri, Jun 2, 2023 at 11:47 AM vanes ***@***.***> wrote:

> Do you have a dapp I can test with?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2486#discussioncomment-6074895>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIGBQYGK7DW3S3SNDNAJ43XJIYLVANCNFSM6AAAAAAYR6AIGI>
> .
> You are receiving this because you were mentioned.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2486/comments/6074895
> @github.com>
>
-- 
Edward Bishop
Co-Founder and CEO, Kelp
604-889-9831
https://kelp.finance

**Reply by finessevanes** [reply]
Actually, you don't need to. I'm going to test that I can get logged in. We normally test a signing function, but you don't need to send funds.

**Comment by edbishop** [comment]
Sent 0.01BNB

On Fri, Jun 2, 2023 at 12:09 PM vanes ***@***.***> wrote:

> For sure! Here is my address: 0xE3b4E50EA72c02838c24856EAc5E4cE35e20ca79
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2486#discussioncomment-6075027>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIGBQ2FJHQNSYX35DJ6XU3XJI3ABANCNFSM6AAAAAAYR6AIGI>
> .
> You are receiving this because you were mentioned.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2486/comments/6075027
> @github.com>
>
-- 
Edward Bishop
Co-Founder and CEO, Kelp
604-889-9831
https://kelp.finance

**Comment by edbishop** [comment]
No worries. It’s sent. Our current use case is specifically with our dApp.
We know that WCv1 is sunsetting this month and we will upgrade to V2 prior.
We just wanted our users to find the wallet in the web3 modal for our
launch next week so that they can use the mobile browser version of our
dApp.

On Fri, Jun 2, 2023 at 12:11 PM vanes ***@***.***> wrote:

> Actually, you don't need to. I'm going to test that I can get logged in.
> We normally test a signing function, but you don't need to send funds.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2486#discussioncomment-6075043>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIGBQ5CK5AMG53ZFE6F7RTXJI3HVANCNFSM6AAAAAAYR6AIGI>
> .
> You are receiving this because you were mentioned.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2486/comments/6075043
> @github.com>
>
-- 
Edward Bishop
Co-Founder and CEO, Kelp
604-889-9831
https://kelp.finance

**Reply by finessevanes** [reply]
OH - thank you. Should have deleted the message faster.

Okay, I'm good to approve your submission, but because we asked for changes it doesn't appear as I can just approve. Could you submit a minor change (which you can immediately revert) to move it back to "Pending Review"? Apologies for the additional step.

**Comment by edbishop** [comment]
Will do!  Thank you!

On Fri, Jun 2, 2023 at 12:20 PM vanes ***@***.***> wrote:

> OH - thank you. Should have deleted the message faster.
>
> Okay, I'm good to approve your submission, but because we asked for
> changes it doesn't appear as I can just approve. Could you submit a minor
> change (which you can immediately revert) to move it back to "Pending
> Review"? Apologies for the additional step.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2486#discussioncomment-6075091>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIGBQ6O6TKEJ4YVYMULVV3XJI4JBANCNFSM6AAAAAAYR6AIGI>
> .
> You are receiving this because you were mentioned.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2486/comments/6075091
> @github.com>
>
-- 
Edward Bishop
Co-Founder and CEO, Kelp
604-889-9831
https://kelp.finance

**Comment by edbishop** [comment]
Running into an error.



On Fri, Jun 2, 2023 at 12:20 PM vanes ***@***.***> wrote:

> OH - thank you. Should have deleted the message faster.
>
> Okay, I'm good to approve your submission, but because we asked for
> changes it doesn't appear as I can just approve. Could you submit a minor
> change (which you can immediately revert) to move it back to "Pending
> Review"? Apologies for the additional step.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2486#discussioncomment-6075091>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIGBQ6O6TKEJ4YVYMULVV3XJI4JBANCNFSM6AAAAAAYR6AIGI>
> .
> You are receiving this because you were mentioned.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2486/comments/6075091
> @github.com>
>
-- 
Edward Bishop
Co-Founder and CEO, Kelp
604-889-9831
https://kelp.finance

**Reply by finessevanes** [reply]
An error on the Cloud side?

**Comment by edbishop** [comment]
Yes. That is what happens when I click on the “Make Changes” button and
attempt to move the listing back to a draft.

On Fri, Jun 2, 2023 at 12:26 PM vanes ***@***.***> wrote:

> An error on the Cloud side?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2486#discussioncomment-6075131>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIGBQ66JV77CZGDQNURNL3XJI46DANCNFSM6AAAAAAYR6AIGI>
> .
> You are receiving this because you were mentioned.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2486/comments/6075131
> @github.com>
>
-- 
Edward Bishop
Co-Founder and CEO, Kelp
604-889-9831
https://kelp.finance

**Reply by finessevanes** [reply]
I don't see any images in the replies above.

**Comment by edbishop** [comment]
Ah.

I’m getting the following error:

Updating the listing failed: “new row violates row-level security policy
for table \”registry_submission\””

On Fri, Jun 2, 2023 at 12:30 PM vanes ***@***.***> wrote:

> I can't see any images.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/WalletConnect/discussions/2486#discussioncomment-6075153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIGBQ5LROZW3SXNVXSZ4PLXJI5M5ANCNFSM6AAAAAAYR6AIGI>
> .
> You are receiving this because you were mentioned.Message ID:
> <WalletConnect/walletconnect-monorepo/repo-discussions/2486/comments/6075153
> @github.com>
>
-- 
Edward Bishop
Co-Founder and CEO, Kelp
604-889-9831
https://kelp.finance

**Reply by finessevanes** [reply]
Thanks, I let our team know.

**Reply by edbishop** [reply]
@finessevanes Any update on the listing error?  Anyway we can push this through before tomorrow?

**Reply by finessevanes** [reply]
Hey there! I've sent another notification to our team. Most of our team members are currently offline due to the CET time zone, but I'll tag a couple of people, @Cali93 and @JeroldG, so they can review it tomorrow when they're available.

**Reply by Cali93** [reply, marked as answer]
Hi @edbishop,

> I’m getting the following error:
> 
> Updating the listing failed: “new row violates row-level security policy
> for table \”registry_submission\””

Thanks for reporting this issue.
It has been fixed a few minutes ago and you should now be able to update your submission.

Sorry for the inconvenience.

**Comment by Cali93** [comment]
Hey @edbishop,

Your submission has been approved.
We're counting on you to migrate to WalletConnect v2 as soon as possible. 🙏

**Reply by edbishop** [reply]
We have already started on the migration!  Thank you for the acceptance!
